tests/br_tiflash/run.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+#
+# Copyright 2020 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+DB="${TEST_NAME}_DATABASE"
+RECORD_COUNT=1000
+
+
+run_sql "CREATE DATABASE $DB" 
+
+run_sql "CREATE TABLE $DB.kv(k varchar(256) primary key, v int)" 
+run_sql "ALTER TABLE $DB.kv SET TIFLASH REPLICA 1" 
+
+stmt="INSERT INTO $DB.kv(k, v) VALUES ('1-record', 1)"
+for i in $(seq 2 $RECORD_COUNT); do
+    stmt="$stmt,('$i-record', $i)"
+done
+run_sql "$stmt"
+
+rm -rf "/${TEST_DIR}/$DB"
+run_br backup full -s "local://$TEST_DIR/$DB" --pd $PD_ADDR
```

Comment:
Could you check tiflash replication progress before backup? 

https://pingcap.com/docs/stable/tiflash/use-tiflash/#check-the-replication-progress

Reply:
This part is buggy now... Let me check my config...

Reply:
```
Traceback (most recent call last):
  File "flash_cluster_manager.py", line 261, in main
  File "flash_cluster_manager.py", line 139, in __init__
  File "flash_cluster_manager.py", line 152, in _update_cluster
  File "flash_cluster_manager.py", line 79, in compute_cur_store
Exception: Can not tell current store.
service_addr: 127.0.0.1:3932,
all tiflash stores: [{'id': 105, 'address': '127.0.0.1:3930', 'labels': [{'key': 'engine', 'value': 'tiflash'}], 'version': 'v4.1.0-alpha-48-g482f94ae6', 'peer_address': '127.0.0.1:20170', 'status_address': '127.0.0.1:17000', 'git_hash': '482f94ae6b96d282a1ecc5c4d3e84ae226f6e7d4', 'start_timestamp': 1591085657, 'deploy_path': '/home/hillium/GolandProjects/br/bin', 'last_heartbeat': 1591085667839402817, 'state_name': 'Up'}]
```
Why...? :thinking:

Reply:
3932 vs 3930 ...:see_no_evil: